packages/story-editor/src/editorApp.js

```diff
@@ -52,23 +50,21 @@ import { GlobalStyle as CalendarStyle } from './components/form/dateTime/calenda
 import KeyboardOnlyOutlines from './utils/keyboardOnlyOutline';
 import { MetaBoxesProvider } from './integrations/wordpress/metaBoxes';
```

Comment:
Another meta boxes related component that should move to `wp-story-editor`. Just mentioning here so we don't forget it.